Source/WebGPU/WebGPU/RenderBundleEncoder.mm

```diff
@@ -555,8 +555,7 @@ - (instancetype)initWithICB:(id<MTLIndirectCommandBuffer>)icb containerBuffer:(i
 {
     m_currentCommand = nil;
 
-    constexpr auto approximateCommandSize = 512;
-    static const auto maxCommandCount = std::max<uint32_t>(100000, m_device->limits().maxBufferSize / approximateCommandSize);
+    constexpr auto maxCommandCount = 0x4000;
```

Comment:
Does it make sense to make it

```
std::min<uint32_t>(0x4000, m_device->limits().maxBufferSize / approximateCommandSize)
```

Or does the maxBufferSize not matter in this case?

Reply:
In this case it doesn’t matter, since maxBufferSize is always 256MB or larger so 256MB / 512 is 50,000